lib/drivelist.js

```diff
@@ -46,14 +47,14 @@ const scripts = require('./scripts');
  */
 exports.list = (callback) => {
   const operatingSystem = os.platform();
-  const script = scripts.paths[operatingSystem];
+  const script = scripts[operatingSystem];
```

Comment:
minor nit, but if we renamed this from `script` to `script_text` then that might make these changes more obvious? ;)